nlpnet/networkconv.pyx

```diff
@@ -222,8 +226,9 @@ Output size: %d
         nn.transitions = transitions if transitions.shape != () else None
         nn.padding_left = data['padding_left']
         nn.padding_right = data['padding_right']
-        nn.pre_padding = np.array((nn.word_window_size / 2) * [nn.padding_left])
-        nn.pos_padding = np.array((nn.word_window_size / 2) * [nn.padding_right])
+
+        nn.pre_padding = np.array((nn.word_window_size / 2) * np.asarray(nn.padding_left))
```

Comment:
This is not related to your pr, but it might be a good call use numpy mathematical operators http://docs.scipy.org/doc/numpy/reference/routines.math.html. They are optimized